test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -52,45 +52,138 @@ build_branch()
             ./configure $config && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
 }
 
+#############################################################
+# get_config_file_name:
+#       arg1: branch name
+#############################################################
+get_config_file_name()
+{
+    local file_name=""
+    branch_name=$1
+    if [ "${wt_standalone}" = true ] || [ $older = true ] ; then
+        file_name="${branch_name}/test/format/CONFIG_default"
+        echo $file_name
+        return
+    fi
+    release=$(echo $branch_name |  cut -c9-)
```

Comment:
Assuming the `cut` command is to remove the `mongodb-` prefix from the branch name? It does not feel very obvious, can we use the [variable `#` denotation](https://stackoverflow.com/questions/16623835/remove-a-fixed-prefix-suffix-from-a-string-in-bash) instead?

Reply:
Updated the script to use the full branch name and append it to the CONFIG string, ex CONFIG_mongodb-4.2, etc